### DISCUSSION TITLE
How can I change the general fonts?

### DISCUSSION THREAD

**Discussion by Photonico** [marp-team/marp#454] [Q&A] [answered]
How can I change the general fonts?

I do not have any experiments with CSS.

this does not work:
```
---
marp: true
math: katex
size: 16:9
style: 'style.css'
paginate: true
---

<style scoped>
section {
  font-family: "Times New Roman", serif;
}
</style>

<!-- _backgroundImage: URL('Pictures/01_background_luke.png')-->

# Analytical Mechanics

## Lagrange's Equation

___

```

**Comment by yhatt** [comment, marked as answer]
Your CSS definitions in `<style scoped>` are absolutely correct, but it looks like that the wrong usage of style directive makes interrupting to apply inline styles in the whole of Markdown document.

By removing this line, your style should work well:

```diff
 ---
 marp: true
 math: katex
 size: 16:9
-style: 'style.css'
 paginate: true
 ---
```

`style` directive must set the content of CSS styles, and _cannot specify the file name_. Please refer to https://marpit.marp.app/directives?id=tweak-theme-style.
